multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/cf/metadata/entity/processor/MtaMetadataApplicationCollector.java

```diff
@@ -16,4 +16,9 @@
     public List<CloudApplication> collect(CloudControllerClient client, MtaMetadataCriteria criteria) {
         return client.getApplicationsByMetadataLabelSelector(criteria.get());
     }
+
+    @Override
+    public List<CloudApplication> collectWithoutAuxiliaryContent(CloudControllerClient client, MtaMetadataCriteria criteria) {
+        return collect(client, criteria);
```

Comment:
collectWithoutAuxiliaryContent invokes the method which adds an auxiliary content